utils/defaulter.go

```diff
@@ -215,12 +215,15 @@ func (d *Defaulter) getEntitySchema(entityType string) (map[string]interface{},
 		return schema, err
 	}
 	resp, err := d.client.Do(d.ctx, req, &schema)
+	if err != nil {
```

Comment:
I think we want to preserve the desired behavior described here: https://github.com/Kong/deck/pull/904/files#diff-d85c74f21d13d17a0932f304636914d8aacd18acb50f86632aee600dd1080b20R221

looks like we are receiving a `*kong.APIError` with  the 404 code, so we could check this instead of checking the resp.StatusCode.

Reply:
@GGabriele already has PR up for this that I missed (#869). The `Client.Do()` method is hooked up in a bit of an abnormal way, so that threw me for a loop.